src/components/MoneyRequestConfirmationList.js

```diff
@@ -100,6 +104,21 @@ const propTypes = {
     /* Onyx Props */
     /** Collection of categories attached to a policy */
     policyCategories: PropTypes.objectOf(categoryPropTypes),
+
+    /** ID of the transaction that represents the money request */
+    transactionID: PropTypes.string,
+
+    /** Transaction that represents the money request */
+    transaction: transactionPropTypes,
+
+    /** Unit and rate used for if the money request is a distance request */
+    mileageRate: PropTypes.shape({
+        unit: PropTypes.oneOf([CONST.CUSTOM_UNITS.DISTANCE_UNIT_MILES, CONST.CUSTOM_UNITS.DISTANCE_UNIT_KILOMETERS]),
+        rate: PropTypes.number,
+    }),
+
+    /** Whether the money request is a distance request */
+    isDistanceRequest: PropTypes.bool,
```

Comment:
Can't this be checked based on the `transaction` prop?

Reply:
ah yes we can do. `Transaction` object wasn't reset before when switching to different money request type, but now that reset behavior is working.

I can refactor this as a separate PR though after this one is merged (I have to change all the places where I use `selectedTab` to check `isDistanceRequest`